registrations/tests/test_signup_group_patch.py

```diff
@@ -33,202 +41,312 @@ def assert_patch_signup_group(api_client, signup_group_pk, signup_group_data):
-    assert signup0.extra_info is None
-    assert signup1.extra_info == "signup1 extra info"
+    first_signup.refresh_from_db()
+    second_signup.refresh_from_db()
+    assert first_signup.presence_status == SignUp.PresenceStatus.PRESENT
+    assert second_signup.presence_status == SignUp.PresenceStatus.NOT_PRESENT
+    assert first_signup.extra_info is None
+    assert second_signup.extra_info == "signup2 extra info"
 
 
 @freeze_time("2023-03-14 03:30:00+02:00")
 @pytest.mark.django_db
 def test_admin_cannot_patch_signups_data(user_api_client, registration):
     signup_group = SignUpGroupFactory(registration=registration)
-    signup0 = SignUpFactory(signup_group=signup_group, registration=registration)
-    signup1 = SignUpFactory(signup_group=signup_group, registration=registration)
+    first_signup = SignUpFactory(signup_group=signup_group, registration=registration)
+    second_signup = SignUpFactory(signup_group=signup_group, registration=registration)
 
     signup_group_data = {
         "signups": [
-            {"id": signup0.pk, "presence_status": SignUp.PresenceStatus.PRESENT},
-            {"id": signup1.pk, "extra_info": "signup1 extra info"},
+            {"id": first_signup.pk, "presence_status": SignUp.PresenceStatus.PRESENT},
+            {"id": second_signup.pk, "extra_info": "signup2 extra info"},
         ]
     }
 
-    assert signup0.presence_status == SignUp.PresenceStatus.NOT_PRESENT
-    assert signup1.presence_status == SignUp.PresenceStatus.NOT_PRESENT
-    assert signup0.extra_info is None
-    assert signup1.extra_info is None
+    assert first_signup.presence_status == SignUp.PresenceStatus.NOT_PRESENT
+    assert second_signup.presence_status == SignUp.PresenceStatus.NOT_PRESENT
+    assert first_signup.extra_info is None
+    assert second_signup.extra_info is None
 
     response = patch_signup_group(user_api_client, signup_group.id, signup_group_data)
     assert response.status_code == status.HTTP_403_FORBIDDEN
 
-    signup0.refresh_from_db()
-    signup1.refresh_from_db()
-    assert signup0.presence_status == SignUp.PresenceStatus.NOT_PRESENT
-    assert signup1.presence_status == SignUp.PresenceStatus.NOT_PRESENT
-    assert signup0.extra_info is None
-    assert signup1.extra_info is None
+    first_signup.refresh_from_db()
+    second_signup.refresh_from_db()
+    assert first_signup.presence_status == SignUp.PresenceStatus.NOT_PRESENT
+    assert second_signup.presence_status == SignUp.PresenceStatus.NOT_PRESENT
+    assert first_signup.extra_info is None
+    assert second_signup.extra_info is None
 
 
 @freeze_time("2023-03-14 03:30:00+02:00")
 @pytest.mark.django_db
-def test_regular_user_cannot_patch_signups_data(user_api_client, registration, user):
-    default_org = user.get_default_organization()
-    default_org.regular_users.add(user)
-    default_org.admin_users.remove(user)
+def test_regular_user_cannot_patch_signups_data(api_client, registration):
+    user = UserFactory()
```

Comment:
In other tests term regular user is used for an user who is a regular users in the publisher organization. 

e.g.
```
user = UserFactory()
user.organization_memberships.add(event.publisher)
```